docs/cli/build/in-obs.md

```diff
@@ -0,0 +1,53 @@
+---
+title: in-obs
+date: 2024-06-27
+sidebar_position: 5
+---
+
+# `packit build in-obs`
+
+Submit a build of the present content in the upstream or local repository to  [OBS](https://build.opensuse.org/).
+
+## Requirements
+
+* SPEC file for the project placed in the repository.
+* Packit config file placed in the repository.
+
+
+
+## Tutorial
+
+1. [Place a config file for packit in the root of your upstream repository.](/docs/configuration/)
+
+2. The command below would create a SRPM from the present content of a repo and perform `osc commit` with it. 
```

Comment:
The srpm creation is an internal detail that is invisible to the user. It's more important to say that it creates a tarball/archive of your repository and commits it along with the spec file to OBS into a package with a name corresponding to your upstream project into the project `home:$username:packit`. Running `packit build in-obs` a second time, creates a new commit in that package.